docs/source/using/experimental-features.adoc

```diff
@@ -87,3 +87,31 @@ PS> minishift.exe start `
 Be sure to specify a valid gateway and nameserver.
 Failing to do so will result in connectivity issues.
 ====
+
+
+[[set-fixed-ip]]
+== Set fixed IP Address
+
+While the VM gets assigned an IP address using DHCP, most hypervisors do not handle extending the lease time of the assignment.
+This can cause issues on restarts which assigns a new IP address to the instance. While the VM will start, it will conflict
+with the security certificates that have been generated before and the deployment will fail.
+
+For this purpose, we have added a functionality which can set a static address to the VM. This will prevent it from changing
+between restarts. However, it will not work on all the driver plugins at the moment due to the way the IP address is resolved.
+
+The following command will set the IP address that was assigned as fixed:
+
+----
+$ minishift ip --set-static 
+----
+
+If for some reason you prefer to use a dynamic assignment, you can use:
+
+----
+$ minishift ip --set-dhcp
+----
+
+[NOTE]
+====
+Currently not supported on KVM as the driver plugin relies on the DHCP offer to determine the IP address.
```

Comment:
I feel like this information should come at the very beginning of the section rather than at the end of it. If you're using KVM, you could end up reading this entire section before realizing that it doesn't apply to your setup.

Reply:
will move this up